tapir/shifts/templates/shifts/shift_detail.html

```diff
@@ -24,10 +24,12 @@
 {% block content %}
     <div class="card mb-2" id="shift_detail_card">
         <div class="card-header">
-            <h5 class="d-flex justify-content-between align-items-center">
-                <span id="shift_card_title">{% translate 'Shift' %}: {{ shift.get_display_name }}</span>
+            <div class="d-flex justify-content-between align-items-center">
+                <h5>
+                    <span id="shift_card_title">{% translate 'Shift' %}: {{ shift.get_display_name }}</span>
+                </h5>
                 {% if perms.shifts.manage %}
-                    <span>
+                    <span class="d-flex flex-row-reverse flex-wrap gap-2">
```

Comment:
I think `justify-content-end` is more appropriate than `flex-flow-reverse` since it keeps the button order.

Reply:
Yes. And I insert also table-responsive for the slot-list-element.